Apps/WebClient/src/ClientApp/src/views/timeline.vue

```diff
@@ -209,13 +209,15 @@
                 <LinearTimeline
                     v-show="isListView && !isLoading"
                     :timeline-entries="timelineEntries"
+                    :is-list-view="isListView"
                     :total-entries="getTotalCount()"
                     :filter-text="filterText"
                     :filter-types="filterTypes"
                 />
                 <CalendarTimeline
                     v-show="!isListView && !isLoading"
                     :timeline-entries="timelineEntries"
+                    :is-list-view="isListView"
```

Comment:
instead of the property  named is-list-view it can be just is-visible

Reply:
thanks @FuriousLlama , please can you review my last checkin?